server/aws-lsp-yaml-json/patchYamlJsonPackage.js

```diff
@@ -35,9 +35,13 @@ const filePathToPatchPathUnsafeEval = {
 function applyPatch(filePathToPatchPath) {
     for (var filePath in filePathToPatchPath) {
         const pathToPatch = `${__dirname}/${filePathToPatchPath[filePath]}`
-        const script = `cd ${rootPackage} && patch ${filePath} ${pathToPatch}`
-        console.log(script)
-        const output = execSync(script, { encoding: 'utf-8', timeout: 2000 })
+        const patchProc = execFile('patch', [filePath, pathToPatch], { cwd: rootPackage, encoding: 'utf-8', timeout: 2000 })
```

Comment:
fyi: if you want sync there is also `execFileSync`

Reply:
Fixed in the latest version.